frontend/src/views/Plaza.vue

```diff
@@ -149,6 +167,12 @@
                   <h2 :class="powerLimitExceeded ? 'text-danger' : ''">{{$t('CharacterDisplay.power')}} +{{burnPower}}/{{remainingPowerLimit}}</h2>
                 </div>
               </div>
+              <div v-if="isTransferring">
+                <b-form-input placeholder="Receiver address" v-model="receiverAddress"/>
+                <div class="transferResultContainer">
+                  <span class="resultMsg text-center"> {{(!isValidWeb3Address(receiverAddress) && receiverAddress !== '') ? 'Invalid address' : ''}} </span>
```

Comment:
isValidWeb3Address might also check for empty string, please check, if not, leave it as-is. Missing translation "Invalid address"

Reply:
I wouldn't expect it to consider empty as valid.  The empty check was modeled on other code that did that check.